setup.py

```diff
@@ -11,6 +11,7 @@
 
 # Dependencies of GPflow
 requirements = [
+    'gast==0.2.2',
```

Comment:
`gast` is a third party library, and this dependency should be resolved by `tfp` probably. Why did you put it here with this version?